install-dev/data/xml/tab.xml

```diff
@@ -5,450 +5,697 @@
                 <class_name>AdminParentModulesSf</class_name>
+                <wording>Modules</wording>
+                <wording_domain>Admin.Navigation.Menu</wording_domain>
             </tab>
                 <tab id="Module_Page" id_parent="Modules" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminModulesSf</class_name>
+                    <wording>Module Manager</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                     <tab id="Module_Page_Manage" id_parent="Module_Page" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminModulesManage</class_name>
+                        <wording>Modules</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                     <tab id="Module_Page_Notifications" id_parent="Module_Page" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminModulesNotifications</class_name>
+                        <wording>Alerts</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                     <tab id="Module_Page_Updates" id_parent="Module_Page" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminModulesUpdates</class_name>
+                        <wording>Updates</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                 <tab id="Module_Catalog" id_parent="Modules" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminParentModulesCatalog</class_name>
+                    <wording>Module Catalog</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                     <tab id="Module_Page_Catalog" id_parent="Module_Catalog" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminModulesCatalog</class_name>
+                        <wording>Module Catalog</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                     <tab id="Marketing_tools" id_parent="Module_Catalog" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminAddonsCatalog</class_name>
+                        <wording>Module Selections</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                 <tab id="Module_Page_old" id_parent="Modules" active="0" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminModules</class_name>
+                    <wording/>
+                    <wording_domain/>
```

Comment:
Normal ?

Reply:
Yes, this tab was never translated. I don't really know why it's here, notice it's `active="0"`.

Reply:
Ok for me ;)